Calculation.py

```diff
@@ -0,0 +1,18 @@
+import output as op
+import spouseOrParentShares as sps
+
+def calculate(amount,sons,daughters,mother,father,wife):
```

Comment:
Is `calculate` a good function name? A function should through its signature and docstrings that what's going on there.